tests/ctl/core/test_api_helpers.py

```diff
@@ -32,9 +32,9 @@ def created_resources(
     resource_type = request.param
     for _ in range(RESOURCE_CREATION_COUNT):
         base_resource = resources_dict[resource_type].model_copy()
-        base_resource.fides_key = "{}_{}".format(
-            base_resource.fides_key, str(uuid.uuid4())[:6]
-        )
+        uuid_suffix = str(uuid.uuid4())[:6]
+        base_resource.fides_key = "{}_{}".format(base_resource.fides_key, uuid_suffix)
+        base_resource.name = "{} {}".format(base_resource.name, uuid_suffix)
```

Comment:
We check name now for uniqueness, so I just added the UUID to the end of the name